src/hooks/useQuery.spec.ts

```diff
@@ -130,7 +130,6 @@ describe('useQuery', () => {
     `;
```

Comment:
Think I saw a comment in an email from you @andyrichardson, but yep, to explain the naming behind this file, I basically wanted a way to differentiate it from `useQuery.test.tsx`. I'd be happy to use a different name. Part of the reason for bringing in `react-hooks-testing-library` was to make us less dependent on `enzyme`, to avoid re-assignment of let bindings in tests, _and_ to be able to get the direct values returned by our hooks so we can assert on them (which I think has some value). Perhaps we could specify these two files as `useQuery.enzyme.test.tsx` and `useQuery.rhtl.test.tsx`? I'm also fine with removing them if we don't think they have value or we don't like the API, I just find the it a bit easier to work w/ than `enzyme`.

Reply:
Cheers @parkerziegler - I initially asked but then found your PR with the explanation. No need to make changes 👍

I'm pretty sure `react-hooks-testing-library` is using react-test-renderer under the hood (we're not using enzyme). I agree the reassignment + added complexity is a pain 😑